2-AAD-Migration/2-2-Security-Groups/README.md

```diff
@@ -0,0 +1,70 @@
+# Migrating AD FS Security Group to Azure Active Directory
+
+> This sample assumes that you already have an AD FS environment.
+
+## Scenario
+
+You have an AD FS application that uses security groups, and you want that applications on Azure AD can work with the same groups.
+
+## About the sample
+
+This documentation guides you how to configure an Azure AD application to include the security groups used on an AD FS application.
+
+### Pre-requisites
+
+- An AD FS environment
+- An Azure Active Directory (Azure AD) tenant. For more information on how to get an Azure AD tenant, see [How to get an Azure AD tenant](https://azure.microsoft.com/en-us/documentation/articles/active-directory-howto-tenant/)
+- [Azure AD Connect](https://docs.microsoft.com/azure/active-directory/hybrid/how-to-connect-sync-whatis) configured on a domain joined machine
+
+## Migrate Security Groups
+
+Users and groups will get synced to Azure AD using the Azure AD Connect tool out of the box, as long as they are presented in the **Synced OU** folder on AD FS.
+
+If you haven't set Azure AD Connect tool yet, please refer to [chapter 1-2-Setup-AzureADConnect](https://github.com/Azure-Samples/ms-identity-dotnet-adfs-to-aad/tree/master/1-ADFS-Host/1-2-Setup-AzureADConnect) first.
+
+## Configure Security Group Claims
+
+1. Sign in to the [Azure portal](https://portal.azure.com).
+2. Navigate to the Microsoft identity platform for developers [App registrations](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/RegisteredApps) page.
+3. Search and select the application where you want to include Security Groups. For instance, `WebApp_SAML`.
```

Comment:
Was that supposed to be the same 'WebApp_SAML' from previous tutorial created via 'Enterprise Applications'? I though  it would show up on 'App Registrations' as well but apparently not.